index.js

```diff
@@ -145,11 +157,33 @@ function randomCase(input) {
   return convert(input, () => Math.round(Math.random()));
 }
 
+/**
+ * Converts matched letters to a specified case
+ * @param {string} input - string to convert
+ * @param {(string | RegExp)} regex - use to match letters you want to convert
+ * @caseType {string} caseType - the case type
+ * @return the given input with match letters to the  case type determined
+ */
+function convertToSpecificCase(input, regex, caseType) {
+  if (caseType !== "lower" && caseType !== "upper") {
```

Comment:
I think is more clear if `str => str.toUpperCase()` or `str => str.toLowerCase()` will be passed as the 3rd argument. And named like `replaceCallback` or [`replacer`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/String/replace#Specifying_a_function_as_a_parameter) and there is no need to check for argument but just `return input.replace(regex, replaceCallback);`